inception-imls-stringmatch/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/stringmatch/StringMatchingRecommender.java

```diff
@@ -247,74 +243,34 @@ public EvaluationResult evaluate(List<CAS> aCasses, DataSplitter aDataSplitter)
         }
 
         // Predict
-        List<Sample> actualData = new ArrayList<>();
+        List<AnnotatedTokenPair> predictions = new ArrayList<>();
```

Comment:
I think "predictions" is misleading here since the data also includes the gold labels.